examples/apps/ai-collab/src/components/UserProfilePhoto.tsx

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InteractiveBrowserCredential } from "@azure/identity";
+import { Client } from "@microsoft/microsoft-graph-client";
+import { TokenCredentialAuthenticationProvider } from "@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials";
+import { Avatar, Badge } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+
+import type { UserPresence } from "@/app/presence";
+
+interface UserProfilePhotoProps {
+	onlineUsers: UserPresence;
+}
+
+const UserProfilePhoto: React.FC<UserProfilePhotoProps> = ({ onlineUsers }) => {
+	const [photos, setPhotos] = useState<string[]>([]);
```

Comment:
Nit: can we name this `photoUrls` and `setPhotoUrls`? In their current form they make me think they contain the actual images, and I got a bit confused to see that they're strings.

Reply:
Updated